docs/concepts/models.md

```diff
@@ -695,6 +695,41 @@ Here are some additional notes on the behavior of [`model_construct()`][pydantic
     not stored in `__pydantic_extra__` or `__dict__` on the instance.
     * Unlike when instantiating the model with validation, a call to [`model_construct()`][pydantic.main.BaseModel.model_construct] with [`extra`][pydantic.ConfigDict.extra] set to `'forbid'` doesn't raise an error in the presence of data not corresponding to fields. Rather, said input data is simply ignored.
 
+## Model copy
```

Comment:
Moved from the serialization page, I don't think model copies have any relation with serialization.